requirements.txt

```diff
@@ -3,3 +3,4 @@
 # SPDX-License-Identifier: Unlicense
 
 adafruit-circuitpython-display-text
+adafruit-blinka
```

Comment:
I'm not sure it really matters, but for the sake of consistency, please capitalise `Adafruit-Blinka`.
```suggestion
Adafruit-Blinka
```

Reply:
Roger roger, updated!